### ISSUE TITLE
automatic facility for belief space discretization

### ISSUE THREAD

**Issue by kach** [kach/memo#39]
to model POMDPs

imagining a special operator that takes something Alice doesn't know, and maps her belief over it to an element of a domain that models the belief space

**Comment by chentoast** [comment]
Hoping to do this + belief filtering sometime soon

**Comment by kach** [comment]
Yes! It would be good to have belief-related utilities collected in memo/utils. For discretization, I think we should do stars & bars. For n belief "tokens" and k hypotheses there should be \binom{n + k - 1}{k - 1} points in the belief space.

**Comment by kach** [comment]
Did you get a chance to look at this, Tony?

**Comment by chentoast** [comment]
I put up a draft pr that i can polish tomorrow.
